dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/DataManager.kt

```diff
@@ -359,4 +207,12 @@ class DataManager(
             throw e
         }
     }
+
+    /**
+     * This method removes an entry from the metaDataInformation in-memory storage
+     * @param dataId the dataId of the meta data to be removed from the in-memory store
+     */
+    fun removeEntryFromMetaDataInMemoryStore(dataId: String) {
+        publicDataInMemoryStorage.remove(dataId)
+    }
```

Comment:
Das ist kein meta-data-in-memory-storage, sondern publicDataInMemoryStorage.
Also da liegt das dataset drin, nicht das meta-dataset.

Reply:
Ja das ist korrekt. Benennung ist korrigiert